### ISSUE TITLE
Changing values not possible (eg. ess_acpowersetpoint)

### ISSUE THREAD

**Issue by cm2962** [sfstar/hass-victron#54]
I see changes when I act in VRM. 

But changes in your integration make values always jumping back to zero. Adjustments are not possible:

![image](https://user-images.githubusercontent.com/75210597/222380959-ce80441a-b741-4a81-8b74-668b5d32d672.png)

**Comment by pos-ei-don** [comment]
This is a security feature of victron. You have to reset the value every couple of seconds (60s i think).
So if you want to use this make shure to enable a trigger to repeat setting this value.

**Comment by michaelkuty** [comment]
nope, I think its fixed in https://github.com/sfstar/hass-victron/pull/50

**Comment by cm2962** [comment]
Thank you Michael. What do I need to to to get this up and running?

**Comment by sfstar** [comment]
Install v0.0.9 that was just released.
If it resolves your issue feel free to close this issue

**Comment by cm2962** [comment]
Hi sfstar, first of all many thanks. 

Unfortunately, it does not work. Same behavior as before. 

Any changes I make result in "0". 

![image](https://user-images.githubusercontent.com/75210597/223225078-96a769a6-d281-4ca4-a272-94649b545ff0.png)

**Comment by angyp** [comment]
I had a similar issue. In the integration configuration did you set the current? If not you will need to so the min and maximum values can be setup.
